add player velocity to coords module

Zazcallabah opened the pull request:
Hi, I added player velocity to the coords module, both as vector components and absolute value.

I did it mostly because i wanted it for myself, but if it is something you want for the full mod, here is a pull request.

If you dont think it fits, just ignore this. :)

Zazcallabah committed:
add player velocity to coords module

TreyRuffy commented:
I just added the translations to [Crowdin](https://crowdin.com/project/betterf3/). Once I get some more translations in, I can merge this.

Zazcallabah committed:
get velocity from root vehicle
for mor accurate numbers when in a minecart

display velocity in meters/second instead of meters/tick

Zazcallabah commented:
after testing for a few weeks, I changed it to display meters/s instead of meters/tick, made for more intuitive numbers. I also changed it to read from root vehicle instead of camera entity, which made the reported minecart speed make more sense.

TreyRuffy commented:
Nice job! Thank you for the PR! I will be adding it to 1.20.2 and 1.20.4 soon. I may backport it if I have enough time!
